docs/miniCore.md

```diff
@@ -0,0 +1,194 @@
+# A Minimal Core and Clearly Defined Extension Points
+
+This proposal aims to define a minimal core and sensible extension points.
+
+
+## Consensus on Annotation Syntax
+
+December 2023 we found a possible position around the annotation syntax (sometimes also called shorthand syntax) 
+with added naming to support multi-edges:
+
+```turtle
+:s :p :o {| :u :v |}          # an `spo` edge, implicitly named by a blank node
```

Comment:
Too much white space inside the annotation, and I'm not a fan of adding detail like prefix declarations when the example itself is very clearly just an example and should concentrate on what is new or different.

Reply:
Please either include the prefix declaration, or do something like `ex:s` instead of `:s`. It is *VERY* easy for people to read examples as fully-baked content, which these are not, and that leads to all sorts of misunderstandings.

As to the whitespace, it's intended to make the columns align visually, so it's easy to see the differences and progressions between lines. *You* know what's intended, so it seems like extraneous whitespace. *I* and others are trying to understand what you intended, and this syntactically optional but hopefully permitted whitespace helps with that.

Reply:
I consider all three suggestions - adding a prefix declaration, adding 'ex:' to every term and adding extra whitespace - as superfluous and distracting from the content of the example. However is lured by such terseness into thinking this was actual working RDF lacks the most basic understanding of Turtle and is not the intended audience of this post. We can't speak to everybody all the time.

Reply:
Sloppiness begets sloppiness. Failing to include all necessary pieces of a syntax in development discussion often leads to misunderstanding and/or forgetting about some impacts on or impacts of those omitted pieces of syntax. This can lead to entirely broken "final" documents that need radical revision when the time comes to request a status change to Candidate Recommendation or Proposed Recommendation. I have been in groups where this has happened. It is my hope to never again experience anything similar.

Reply:
I will not engage in this discussion any more and I would ask you to respect my decision and concentrate on discussing what I wrote, not on the style in which I wrote it.